### DISCUSSION TITLE
feature: allow for overriding keymaps of dial.nvim

### DISCUSSION THREAD

**Discussion by quangdn42** [LazyVim/LazyVim#4058] [Q&A]
### Did you check the docs?

- [X] I have read all the LazyVim docs
- [X] This is not a request for a new or existing extra (see above)

### Is your feature request related to a problem? Please describe.

I'm continuing my quest of updating all the default keymaps to match my layout :)

I want to update the keymap of dial.nvim. Here's what I have done in my config
<img width="1192" alt="image" src="https://github.com/user-attachments/assets/1d071e7d-9e2d-4a9b-9bbb-53972981bdfd">

This seems to work inconsistently. Most notably, in lua itself toggling between true and false does not work.

### Describe the solution you'd like

It would be nice if I can simply override the keymaps similar to other plugins. 

### Describe alternatives you've considered

I tested updating the keymaps directly in the source and everything works well, so I'm sure the above config caused the issue.

**Comment by folke** [comment]
Questions belong in discussions. **not** issues/features.
